packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -320,4 +320,31 @@ describe('Date Picker', () => {
 
     Terra.should.validateElement('default', { axeRules: { rules: ignoredA11y } });
   });
+
+  describe('DatePickerField', () => {
+    describe('Valid DatePickerField', () => {
+      before(() => {
+        browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-field');
+      });
+
+      Terra.should.validateElement({ axeRules: { rules: ignoredA11y } });
+    });
+
+    describe('Invalid DatePickerField', () => {
+      before(() => {
+        browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-field');
+        browser.click('#validity-toggle');
+      });
+
+      Terra.should.validateElement({ axeRules: { rules: ignoredA11y } });
```

Comment:
With this new component, can we remove the disabled axe rule `label: { enabled: false },`?

Reply:
I don't think that the DatePickerField component should be used in the DatePicker tests, but they could be updated to have aria-labels to be able to re-enable that rule.

Reply:
Actually, looking at the test failures when I enable that rule, there a11y failures are within the date picker calendar, for the year and month select components. We should probably log a separate issue for that.

Reply:
@jmsv6d ah makes sense